WhyNotWin11.au3

```diff
@@ -509,93 +509,52 @@ Func Main()
-						GUICtrlSetData($hCheck[2][0], "OK")
-						GUICtrlSetBkColor($hCheck[2][0], 0x4CC355)
-						GUICtrlSetData($hCheck[2][2], _Translate($iMUI, "Listed as Compatible"))
-						ExitLoop
-				EndSelect
-			Next
+			$ListFile = "\WhyNotWin11\SupportedProcessorsAMD.txt"
 		Case StringInStr(_GetCPUInfo(2), "Intel")
-			$iLines = _FileCountLines(@LocalAppDataDir & "\WhyNotWin11\SupportedProcessorsIntel.txt")
-			If @error Then
-				GUICtrlSetData($hCheck[2][0], "?")
-				GUICtrlSetBkColor($hCheck[2][0], 0xF4C141)
-				GUICtrlSetData($hCheck[2][2], _Translate($iMUI, "Unable to Check List"))
-			EndIf
-			For $iLine = 1 To $iLines Step 1
-				$sLine = FileReadLine(@LocalAppDataDir & "\WhyNotWin11\SupportedProcessorsIntel.txt", $iLine)
-				Select
-					Case @error = -1
-						GUICtrlSetData($hCheck[2][0], "?")
-						GUICtrlSetBkColor($hCheck[2][0], 0xF4C141)
-						GUICtrlSetData($hCheck[2][2], _Translate($iMUI, "Error Accessing List"))
-						ExitLoop
-					Case $iLine = $iLines
-						GUICtrlSetData($hCheck[2][0], "?")
-						GUICtrlSetBkColor($hCheck[2][0], 0xF4C141)
-						GUICtrlSetData($hCheck[2][2], _Translate($iMUI, "Not Currently Listed as Compatible"))
-						ExitLoop
-					Case StringInStr(_GetCPUInfo(2), $sLine)
-						GUICtrlSetData($hCheck[2][0], "OK")
-						GUICtrlSetBkColor($hCheck[2][0], 0x4CC355)
-						GUICtrlSetData($hCheck[2][2], _Translate($iMUI, "Listed as Compatible"))
-						ExitLoop
-				EndSelect
-			Next
+			$ListFile = "\WhyNotWin11\SupportedProcessorsIntel.txt"
 		Case StringInStr(_GetCPUInfo(2), "SnapDragon") Or StringInStr(_GetCPUInfo(2), "Microsoft")
-			$iLines = _FileCountLines(@LocalAppDataDir & "\WhyNotWin11\SupportedProcessorsQualcomm.txt")
-			If @error Then
-				GUICtrlSetData($hCheck[2][0], "?")
-				GUICtrlSetBkColor($hCheck[2][0], 0xF4C141)
-				GUICtrlSetTip($hCheck[2][0], _Translate($iMUI, "Unable to Check List"))
-			EndIf
-			For $iLine = 1 To $iLines Step 1
-				$sLine = FileReadLine(@LocalAppDataDir & "\WhyNotWin11\SupportedProcessorsQualcomm.txt", $iLine)
-				Select
-					Case @error = -1
-						GUICtrlSetData($hCheck[2][0], "?")
-						GUICtrlSetBkColor($hCheck[2][0], 0xF4C141)
-						GUICtrlSetData($hCheck[2][2], _Translate($iMUI, "Error Accessing List"))
-						ExitLoop
-					Case $iLine = $iLines
-						GUICtrlSetData($hCheck[2][0], "?")
-						GUICtrlSetBkColor($hCheck[2][0], 0xF4C141)
-						GUICtrlSetData($hCheck[2][2], _Translate($iMUI, "Not Currently Listed as Compatible"))
-						ExitLoop
-					Case StringInStr(_GetCPUInfo(2), $sLine)
-						GUICtrlSetData($hCheck[2][0], "OK")
-						GUICtrlSetBkColor($hCheck[2][0], 0x4CC355)
-						GUICtrlSetData($hCheck[2][2], _Translate($iMUI, "Listed as Compatible"))
-						ExitLoop
-				EndSelect
-			Next
+			$ListFile = "\WhyNotWin11\SupportedProcessorsQualcomm.txt"
 		Case Else
+			$ListFile = "error"
+	EndSelect
+	
+	If $ListFile = "error" Then
+		GUICtrlSetData($hCheck[2][0], "X")
+		GUICtrlSetData($hCheck[2][2], _Translate($iMUI, "Unknown CPU"))
```

Comment:
Any CPU that's not AMD, Intel, or Qualcomm is likely incompatible unless I there's documentation proving otherwise. Please use `Not Currently Listed as Compatible`